Autopilot/boardfiles/Src/i2c.c

```diff
@@ -33,7 +33,7 @@ void MX_I2C1_Init(void)
 {
 
   hi2c1.Instance = I2C1;
-  hi2c1.Init.Timing = 0x6000030D;
+  hi2c1.Init.Timing = 0x20404768;
```

Comment:
Comment please?

Reply:
This is an autogenerated value and the comments will get overriden if we change any part of the i2c interface. I would actually suggest that we store the settings used for sensors on a document. Will bring up at tomorrow's meeting